kafka-ui-react-app/src/components/Wizard/WizardForm/SchemaRegistry/SchemaRegistry.tsx

```diff
@@ -5,104 +5,117 @@ import Input from 'components/common/Input/Input';
 import { FormError } from 'components/common/Input/Input.styled';
 import { ErrorMessage } from '@hookform/error-message';
 import { useFormContext } from 'react-hook-form';
+import Heading from 'components/common/heading/Heading.styled';
+import { InputLabel } from 'components/common/Input/InputLabel.styled';
 
 const SchemaRegistry = () => {
-  const [newSchemaRegistry, setNewSchemaRegistry] = useState(false);
-  const methods = useFormContext();
-  const registry: React.MouseEventHandler<HTMLButtonElement> = (e) => {
+  const [isShowRegistryForm, setIsShowRegistryForm] = useState(true);
```

Comment:
Just watch `schemaRegistry` field. If it is empty(undefined) show nothing

Reply:
I didn't understand very well what you mean,
something like this ?

  const [isShowRegistryForm, setIsShowRegistryForm] = useState(!watch('schemaRegistry'));

Reply:
i think what he meant is that `const isShowRegistryForm = watch('schemaRegistry')`, cause as far as it is directly related to the `schemaRegistry` field , so you don't need extra setter on the that level.

Reply:
we need not only reset the form, also we need to close form with clicking button, so just watch schemaRegistry i think not enough.

Reply:
yeah that might not work during non empty default values.